server/cluster/cluster.go

```diff
@@ -595,6 +596,27 @@ func (c *RaftCluster) HandleStoreHeartbeat(stats *pdpb.StoreStats) error {
 	return nil
 }
 
+// processBucketHeartbeat update the bucket information.
+func (c *RaftCluster) processBucketHeartbeat(buckets *metapb.Buckets) error {
+	c.RLock()
+	region := c.core.GetRegion(buckets.GetRegionId())
+	c.RUnlock()
+	if region == nil {
+		return errors.Errorf("region %v not found", buckets.GetRegionId())
```

Comment:
We should define a new error in `pkg/errs`.

Reply:
fix